glad/generator/rust/templates/impl.rs

```diff
@@ -76,7 +97,25 @@ mod storage {
     {{ template_utils.protect(command) }} pub static mut {{ command.name|no_prefix }}: FnPtr = FnPtr { ptr: FnPtr::not_initialized as *const raw::c_void, is_loaded: false };
     {% endfor %}
 }
+{% endif %}
+
+{% if options.mx %}
+pub fn load<F>(mut loadfn: F) -> functions::Gl where F: FnMut(&'static str) -> *const raw::c_void {
+    let gl = Gl {
```

Comment:
I told you this doesn't need to be mut, sorry in fact it has to be `mut` because of the later `aliased` calls, but we need to silence the warning in case there are no aliased functions